ui/chatview.go

```diff
@@ -315,6 +317,11 @@ func (chatView *ChatView) addMessageInternal(message *discordgo.Message) {
 //AddMessage add an additional message to the ChatView.
 func (chatView *ChatView) AddMessage(message *discordgo.Message) {
 	wasScrolledToTheEnd := chatView.internalTextView.IsScrolledToEnd()
+	t1, _ := chatView.data[len(chatView.data)-1].Timestamp.Parse()
+	t2, _ := message.Timestamp.Parse()
+	if equal := times.CompareMessageDates(t1.Local(), t2.Local()); !equal {
```

Comment:
In Go, a function call that returns a boolean is already a valid expression. Therefore:

```go
if booleanState := isX(); !booleanState {
```

is the same as


```go
if !isX() {
```

I am assuming you did that because the name `Compare` doesn't make it obvious that this is actually an equality check. Maybe you should rename the method to `AreTimesTheSameDay` or something like that. What a `Compare` function usually does is checking which of two values is smaller or bigger (or equal).

Reply:
No, earlier the `CompareMessageDates` returned two values. I just forgot to simplify the if statement when i removed the time return

Reply:
I see. Anyway, this is the last thing, after that's resolved, I'll merge :D

Reply:
I think the name is good, i have updated comment to be more accurate